docs/_articles/en/lwc/write-lwc.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Write Lightning Web Components in VS Code
+lang: en
+---
+
+The Lightning Web Components Extension for VS Code leverages the editor's built-in code navigation and language features so that you can efficiently build Lightning web components. 
+
+Take advantage of the following Lightning Web Components Extension for VS Code features:
+- [Code Completion](#code-completion)
+- [View Documentation on Hover](#view-documentation-on-hover)
+- [View Errors Messages on Hover](#view-error-messages-on-hover)
+- [Linting](#linting)
+
+## Code Completion
+
+The Lightning Web Components Extension builds on VS Code's code completion capabilities by completing resource names in HTML and JavaScript files.
+
+VS Code completes static resources. In this example, it completes the `chartjs` object in the `libChartjs` component. `libChartjs` refers to the third-party JavaScript library [chartjs](https://www.chartjs.org/). Include it in your code by importing `loadScript` from the Lightning Web Components `platformResourceLoader` module. Then, add the statement `import chartjs from '@salesforce/resourceUrl/chart';`. Here, VS Code fills in the resource's name when we add it in the import statement.
+
+![Static Resource Completion](./images/vscode_lwc_staticresource.png)
+
+VS Code also completes Lightning web components imports from the `lwc` module. Here, it completes `LightningElement`. When hovered over, `LightningElement` displays information about the `LightningElement` class. 
+
+![LWC Module Completion](./images/vscode_lwc_js.png)
+
+Code completion is also provided for HTML tags and attributes for components in the `lightning` and `c` namespaces. Here, with `c-view-source`, VS Code completes its source attribute.
+
+![HTML Attribute Completion](./images/vscode_lwc_html_attr.png)
+
+To view the source of a `c` namespace component, press **command** + **click**. 
+
+![View Source of c Namespace](./images/vscode_lwc_viewsource.png)
+
+## View Documentation on Hover
+
+VS Code displays documentation when you hover over a standard Lightning web component in the `lightning` namespace. Here is the documentation that displays for `lightning-layout`. The dropdown also provides a link to view the component in the Component Library.
```

Comment:
Its not just for lightning namespace components. It will show documentation also for your custom c-namespace components if you write documentation for them.